mopidy_listenbrainz/listenbrainz.py

```diff
@@ -159,6 +179,11 @@ def list_playlists_created_for_user(self) -> List[PlaylistData]:
                 "Authorization": f"Token {self.token}",
             },
         )
+        try:
+            check_response_status(response)
+        except _RequestError:
+            return []
```

Comment:
I don't think silently failing/with just a warning is the best behavior here.

Reply:
You're right, re-raising would be better.